cmd/opdoc/opdoc.go

```diff
@@ -317,9 +343,10 @@ func argEnums(name string) ([]string, string) {
 }
 
 func buildLanguageSpec(opGroups map[string][]string) *LanguageSpec {
-	opSpecs := logic.OpcodesByVersion(logic.LogicVersion)
-	records := make([]OpRecord, len(opSpecs))
-	for i, spec := range opSpecs {
+	records := make([]OpRecord, len(logic.OpNames))
+	i := 0
+	for _, name := range logic.OpNames {
```

Comment:
Why are you managing `i` "by hand"? I don't see any code that changes it.

Reply:
Good catch, was from a time when OpNames was a map